src/JustSaying/Fluent/PublicationsBuilder.cs

```diff
@@ -136,11 +139,11 @@ public PublicationsBuilder WithTopic<T>(Action<TopicPublicationBuilder<T>> confi
         /// Configures the publications for the <see cref="JustSayingFluently"/>.
         /// </summary>
         /// <param name="bus">The <see cref="JustSayingFluently"/> to configure publications for.</param>
```

Comment:
Either add the missing `///` parameter docs, or just remove all the docs on this method.

Reply:
Done!